torch/_functorch/aot_autograd.py

```diff
@@ -898,6 +898,132 @@ def inner(*flat_args):
 
     return inner
 
+
+@dataclass
+class BackwardSignature:
+    """
+    Provides information about the backward section of an exported
+    joint forward-backward graph.
+    For a particular fx GraphModule, this class contains information on:
+    (1) A mapping from each gradient (backwards output) to the parameter
+        it corresponds to (forward input)
+    (2) A mapping from each gradient (backwards output) to the user input
+        it corresponds to (forward input)
+    (3) Which of the forward outputs corresponds to the loss, that we backprop on.
```

Comment:
worth saying explicitly what the str denotes (node.target, amirite?)

Reply:
`node.name` 😛 yep, will update (it came from here: https://github.com/pytorch/pytorch/pull/100587/files#diff-df954bbf954d2dcb81f687876053267ffa4ddb36ed86b7d2bd76319ff2b94416R3300)